package.json

```diff
@@ -4,12 +4,14 @@
   "description": "Sample App for JavaScript SDK view layer for Onfido identity verification",
   "scripts": {
     "predev": "npm run build",
-    "dev": "NODE_ENV=development webpack-dev-server --inline --hot --progress --https",
+    "dev:webpack": "NODE_ENV=development webpack-dev-server --inline --hot --progress --https --cert cert.pem --key key.pem",
```

Comment:
Do you use these `cert` and `key` args? It looks like you ignore them and use a hardcoded value instead?